### DISCUSSION TITLE
libpag iOS 在高版本系统锁屏进入后台回来渲染空问题

### DISCUSSION THREAD

**Discussion by wangwanjie** [Tencent/libpag#1577] [Q&A]
版本信息】4.2.88 | iOS 16.5
【预期的情况】锁屏后解锁PAGView 渲染画面正常
【实际的情况】锁屏后解锁PAGView会先显示空
【Demo附件】实测用任意 pag 文件都会如此，我日常开发用的是 13.3 的系统，没有此问题。
【复现步骤】打开demo工程真机运行，打开 app 后直接锁屏，再解锁，会看到画面由无到有。
而直接退回桌面再回app无此问题，甚至退回桌面后再锁屏再解锁回app也无此问题。我测试了 16.5 、16.3.1 均有此问题，测试同学用的 15 系统也有这个问题，只有我用的13.3 没这个问题。Demo在附件，请版主有空帮忙看看。
[PAGSceneDemo.zip](https://github.com/Tencent/libpag/files/12117660/PAGSceneDemo.zip)

**Comment by wangwanjie** [comment]
没人理吗。。

**Comment by kevingpqi123** [comment]
初步看了下现象，影响比较有限，这里处理的优先级不会太高

**Reply by wangwanjie** [reply]
可以麻烦提升下优先级吗。。我们有个很重的高曝光页面用了全屏的PAGView，目前日活是W级别，还没全量放开，只是在内测，这个问题非常影响体验

**Reply by kevingpqi123** [reply]
你们可以先通过添加背景图的方式来规避，目前人力有限，个性化的问题还暂时排不上

**Reply by kevingpqi123** [reply]
并且这个问题看起来影响很有限，对于用户的体验影响也不大

**Comment by kevingpqi123** [comment]
影响有限，暂时关闭该 discussion
